.github/ISSUE_TEMPLATE/giskard_bug_template.yaml

```diff
@@ -0,0 +1,136 @@
+name: Giskard Bug Template
+description: Use this template to report any bugs
+body:
+  - type: dropdown
+    id: issue-type
+    attributes:
+      label: Issue Type
+      description: What type of issue would you like to report?
+      multiple: false
+      options:
+        - Bug
+        - Build/Install
+        - Performance
+        - Documentation Bug
+    validations:
+      required: true
+  - type: markdown
+    attributes:
+      value: |
+        Please make sure that this is a bug. As per our [GitHub Policy](https://github.com/Giskard-AI/giskard/blob/main/ISSUES.md),we only address code/doc bugs, performance issues, feature requests and build/installation issues on GitHub.
+
+  # - type: markdown
+  #   attributes:
+      # value: | 
+      #   You can collect some of this information using our environment capture [script](https://github.com/tensorflow/tensorflow/tree/master/tools/tf_env_collect.sh)  You can also obtain the TensorFlow version with: <br> 1. TF 1.0: `python -c "import tensorflow as tf; print(tf.GIT_VERSION, tf.VERSION)"` <br>2. TF 2.0: `python -c "import giskard ; print(tf.version.GIT_VERSION, tf.version.VERSION)"`
```

Comment:
Comment should be deleted

Reply:
As mentioned in the weekly, comments were intentionally kept for @andreybavt to check if some of them can be useful for us.

Reply:
@princyiakov , could you replace this section with a description of where to find giskard server version and giskard python library version:
e.g.
![image](https://user-images.githubusercontent.com/7889985/209945281-2372d73e-4168-4833-aaa7-05da688b3bca.png)
and 
```
import giskard
giskard.__version__
```